src/Host/Client/Impl/Session/RSessionEvaluationCommands.cs

```diff
@@ -153,19 +153,23 @@ public static class RSessionEvaluationCommands {
         public static Task<REvaluationResult> SetVsHelpRedirection(this IRExpressionEvaluator evaluation) {
             var script =
 @"options(help_type = 'html')
-  options(browser = function(url) { 
-      rtvs:::send_message('Browser', url) 
-  })";
+  options(browser = rtvs:::browser)
+";
             return evaluation.EvaluateAsync(script, REvaluationKind.Mutating);
         }
 
-        public static Task<REvaluationResult> SetChangeDirectoryRedirection(this IRExpressionEvaluator evaluation) {
-            var script =
-@"utils::assignInNamespace('setwd', function(dir) {
-    old <- .Internal(setwd(dir))
-    rtvs:::send_message('setwd', dir)
-    invisible(old)
-  }, 'base')";
+        public static Task<REvaluationResult> OverrideFunction(this IRExpressionEvaluator evaluation, string name, string ns) {
+            var script = Invariant($"utils::assignInNamespace('{name}', rtvs:::{name}, '{ns}')");
```

Comment:
`ToRStringLiteral` (unlikely that we'll ever have anything that needs to be escaped here, but you never know).